src/load-config.js

```diff
@@ -33,6 +33,8 @@ if (process.env.ACTUAL_CONFIG_PATH) {
     `loading config from ACTUAL_CONFIG_PATH: '${process.env.ACTUAL_CONFIG_PATH}'`,
   );
   userConfig = parseJSON(process.env.ACTUAL_CONFIG_PATH);
+
+  defaultDataDir = userConfig.dataDir ?? defaultDataDir;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏nice! Could you also add a `debug` line for this new config (in this same file.. scroll down a few lines to find the other `debug` lines).

Reply:
Ah, of course! Updated for you now.